docs/standard/exceptions.md

```diff
@@ -0,0 +1,528 @@
+| [Message](https://docs.microsoft.com/dotnet/core/api/System.Exception#System_Exception_Message) | Provides details about the cause of an exception.
+| [Source](https://docs.microsoft.com/dotnet/core/api/System.Exception#System_Exception_Source) | Gets or sets the name of the application or the object that causes the error. |
+| [StackTrace](https://docs.microsoft.com/dotnet/core/api/System.Exception#System_Exception_StackTrace) | Contains a stack trace that can be used to determine where an error occurred. The stack trace includes the source file name and program line number if debugging information is available. |
+
+Most of the classes that inherit from **Exception** do not implement additional members or provide additional functionality; they simply inherit from **Exception**. Therefore, the most important information for an exception can be found in the hierarchy of exception classes, the exception name, and the information contained in the exception.
+
+It is recommended to throw and catch only objects that derive from **Exception**, but you can throw any object that derives from the [Object](https://docs.microsoft.com/dotnet/core/api/System.Object) class as an exception. Note that not all languages support throwing and catching objects that do not derive from **Exception**.
+
+## Runtime Exceptions
+
+Critical exceptions, such as [OutOfMemoryException](https://docs.microsoft.com/dotnet/core/api/System.OutOfMemoryException), are thrown by the runtime.
+
+The following table lists the standard exceptions that the runtime throws.
+
+| Exception type | Base type | Description | Example |
+| -------------- | --------- | ----------- | ------- |
+| [Exception](https://docs.microsoft.com/dotnet/core/api/System.Exception) | [Object](https://docs.microsoft.com/dotnet/core/api/System.Object) | Base class for all exceptions. | None (use a derived class of this exception). |
```

Comment:
These are largely not runtime exceptions, anyway. I think  that this is more "common exceptions". 

Runtimes exceptions are:

- StackOverflow
- OOM
- ExecutionEngine

So, I think that section as-is is wrong.